<launch>
    
    <node name="vive_calibrate" pkg="vive" type="vive_calibrate" output="screen">
        <rosparam file="$(find vive)/param/vive_calibrate.yaml" command="load" />
    </node>

</launch>